sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/EventGridPublisherClientBuilder.java

```diff
@@ -247,6 +255,19 @@ public EventGridPublisherClientBuilder credential(AzureSasCredential credential)
         return this;
     }
 
+    /**
+     * Set the domain or topic authentication using Azure Activity Directory authentication.
+     * Refer to <a href="https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/identity/azure-identity">azure-identity</a>
```

Comment:
Don't include the GitHub link to azure-identity here.

Reply:
Changed to use this link https://docs.microsoft.com/java/api/overview/azure/identity-readme that has the same content.